libafl/src/executors/inprocess.rs

```diff
@@ -767,8 +809,11 @@ mod windows_exception_handler {
         Z: HasObjective<I, OF, S>,
     {
         // Have we set a timer_before?
-        if let Some(x) = (data.timer_queue as *mut HANDLE).as_mut() {
-            windows_delete_timer_queue(*x);
+        if let Some(x) =
```

Comment:
Is there a race here? Maybe this needs locks too?

Reply:
No crash handler runs in the same thread that triggered the fault, I guess.

Reply:
What if the timer triggers while we're in the crash handler?

Reply:
that's a proble, this needs fix 😀